src/nordjylland_news/summary_data_set.py

```diff
@@ -0,0 +1,135 @@
+import os
+import time
+
+import requests
+
+from nordjylland_news.constants import HEADERS, MAX_PER_PAGE, RAW_DATA_PATH
+from nordjylland_news.utils import append_jsonl, html_to_text, init_jsonl, load_jsonl
```

Comment:
Det er bedst at importere andre moduler direkte fra andre moduler, som relative imports, fremfor at importere dem eksternt som du gør her. Altså burde dette ændres til:

```suggestion
from .constants import HEADERS, MAX_PER_PAGE, RAW_DATA_PATH
from .utils import append_jsonl, html_to_text, init_jsonl, load_jsonl
```

Bemærk dog, at i *scripts* skal vi importere ting eksternt, med absolutte imports. Det er bare når man importerer moduler fra andre moduler, at vi gør det relativt.